sbnanaobj/StandardRecord/SRTrkLikePID.cxx

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRTrkLikePID.cxx
```

Comment:
Personally I first read this as "track-like" even though I know from the context that 'Like' in this situation represents likelihood.

It's a personal choice and I won't hold up the PR for it but would it be worth using the name `SRTrkLikelihoodPID` and branch name `likelihoodpid` to avoid any confusion?